applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_frictionless_components_mortar_contact_condition.cpp

```diff
@@ -53,7 +53,7 @@ Condition::Pointer AugmentedLagrangianMethodFrictionlessComponentsMortarContactC
     PropertiesType::Pointer pProperties,
     GeometryType::Pointer pMasterGeom) const
 {
-    return Kratos::make_intrusive<  AugmentedLagrangianMethodFrictionlessComponentsMortarContactCondition<TDim,TNumNodes, TNormalVariation > >( NewId, pGeom, pProperties, pMasterGeom );
+    return Kratos::make_intrusive<  AugmentedLagrangianMethodFrictionlessComponentsMortarContactCondition<TDim,TNumNodes, TNormalVariation, TNumNodesMaster > >( NewId, pGeom, pProperties, pMasterGeom );
```

Comment:
Out of curiosity, why do you need to template it to the number of nodes of the master?

Reply:
Because I sue bounded matrices for everything (faster and more efficient). yes, I sue a lot of templates